### PR TITLE
load customers block only if a customer was not set

### PR DESCRIPTION
**Pull request by pepijnblom**
As discussed here:   #592 

Sorry this took so long, I kind of forgot about it since my personal solution worked as well but confirmed it worked on another project now.

### CODE REVIEW

File: app/design/adminhtml/default/default/template/sales/order/create/form.phtml [hunk 1]
```diff
@@ -36,7 +36,9 @@
         <?php echo $this->getChildHtml('message') ?>
     </div>
     <div id="order-customer-selector" style="display:<?php echo $this->getCustomerSelectorDisplay() ?>">
-        <?php echo $this->getChildHtml('customer') ?>
+        <?php if (!$this->getCustomerId()): ?>
```
Comment (tmotyl) [inline app/design/adminhtml/default/default/template/sales/order/create/form.phtml]: This condition is an oposite of the commit title.
"don't load customers block if a customer is already set "
Either the commit should read "don't load customers block if a customer is set" or the condition should be chnaged?

**Comment by tmotyl** [comment]
Hi @pepijnblom you don't have to close PR when you want to change code. Just push to the branch and the PR will be automatically updated.

**Comment by pepijnblom** [comment]
> Hi @pepijnblom you don't have to close PR when you want to change code. Just push to the branch and the PR will be automatically updated.

Yeah I get that but I prefer to have as little commits on my merges as possible /ocd 😅

**Comment by tmotyl** [comment]
then rebase the branch, squashing commits together, and push --force. 
It's making reviewing much harder when you open new PR's and the dicscussion history is lost.

**Pull request closed without merging**